libraries/AP_OSD/fonts/HDFonts/WS/font_update.ini

```diff
@@ -0,0 +1,11 @@
+[config]
+count=1
+1=user_ardu
+
+[user_ardu]
+imgname_720=user_ardu_24.png

```

Comment:
These are interesting - how are they used?

Reply:
These files are specific to the Walksanail digital video system, and they should go into the goggles (or VRX) SD card font folder. I decided to include them for completeness, as the files that come "standard" with the system for Ardupilot don't contain the extra set of icons implemented in this PR. But the system accepts custom fonts (unlike DJI), so it's just a matter of replacing the "stock" files with these customized ones. It doesn't affect the actual Ardupilot code at all.